05/01/25 (23:46:13) | ERROR: Exception while getting cpu frequency!
05/01/25 (23:46:13) | ERROR: [Errno 2] No such file or directory (originated from sysctlbyname('hw.cpufrequency'))
Traceback (most recent call last):
  File "/opt/homebrew/bin/bpytop", line 3080, in _collect
    if CONFIG.show_cpu_freq and hasattr(psutil.cpu_freq(), "current"):
  File "/opt/homebrew/Cellar/bpytop/1.0.68/libexec/lib/python3.10/site-packages/psutil/__init__.py", line 1859, in cpu_freq
    ret = _psplatform.cpu_freq()
  File "/opt/homebrew/Cellar/bpytop/1.0.68/libexec/lib/python3.10/site-packages/psutil/_psosx.py", line 180, in cpu_freq
    curr, min_, max_ = cext.cpu_freq()
FileNotFoundError: [Errno 2] No such file or directory (originated from sysctlbyname('hw.cpufrequency'))
09/07/25 (23:29:40) | ERROR: Exception while getting cpu frequency!
09/07/25 (23:29:40) | ERROR: [Errno 2] No such file or directory (originated from sysctlbyname('hw.cpufrequency'))
Traceback (most recent call last):
  File "/opt/homebrew/bin/bpytop", line 3080, in _collect
    if CONFIG.show_cpu_freq and hasattr(psutil.cpu_freq(), "current"):
  File "/opt/homebrew/Cellar/bpytop/1.0.68/libexec/lib/python3.10/site-packages/psutil/__init__.py", line 1859, in cpu_freq
    ret = _psplatform.cpu_freq()
  File "/opt/homebrew/Cellar/bpytop/1.0.68/libexec/lib/python3.10/site-packages/psutil/_psosx.py", line 180, in cpu_freq
    curr, min_, max_ = cext.cpu_freq()
FileNotFoundError: [Errno 2] No such file or directory (originated from sysctlbyname('hw.cpufrequency'))
